.docker/Dockerfile.nginx-drupal

```diff
@@ -14,3 +14,8 @@ COPY --from=cli /app /app
 
 RUN /app/sanitize.sh \
   && rm -rf /app/sanitize.sh
+
+RUN apk add gettext
```

Comment:
OpenResty does not include `gettext` (and so: `envsubst`) by default, though default nginx images do. Required to perform inline variable replacement